packages/polaris-viz/src/hooks/useLinearLabelsAndDimensions.ts

```diff
@@ -55,9 +50,8 @@ export function useLinearLabelsAndDimensions({
     return clamp({
       amount: longestLabelWidth,
       min: 0,
-      max: MAX_LINEAR_LABEL_WIDTH,
```

Comment:
I think the visual change @carysmills noticed was because we removed this line

Reply:
Yup! Going to revert that change.